host/host.go

```diff
@@ -13,6 +13,8 @@ import (
 	"github.com/libp2p/go-libp2p/core/crypto"
 	"github.com/libp2p/go-libp2p/core/host"
 	rcmgr "github.com/libp2p/go-libp2p/p2p/host/resource-manager"
+	relayv2 "github.com/libp2p/go-libp2p/p2p/protocol/circuitv2/relay"
+	webrtc "github.com/libp2p/go-libp2p/p2p/transport/webrtc"
```

Comment:
I don't think we need to alias this since we use the same name.

Reply:
`webrtc` is actually go module as `package libp2pwebrtc`; hence the alias is needed for webrtc; not for relayv2 though as you've stated.

Reply:
https://github.com/blocklessnetwork/b7s/pull/168/commits/d6032bde9d981258a86db1c9545e454c2b540cd1